### PR TITLE
Portfolio-mobile-version-first

### PR DESCRIPTION
**Pull request by cosywasswa**
-  Added new sections(works and about me) to the index.html elements.
-  Styled the html elements for the two sections added including images.
-  Updated the README file to include the two sections added to the project.
-  Added more image folders for the two sections added to the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,210 @@ <h2>I'm a software developer</h2>
           </ul>
         </div>
       </section>
+
+      <section class="grid-container">
+        <div class="recent-works">
+          <h3>My Recent works</h3>
+        </div>
+        <img class="line" src="images/images-self/line (2).png" />
+        <img class="img-works" src="images/images-works/Img Placeholder.png" />
```
Comment (ghost) [inline index.html:50]: - [ ] Good work on your design  but there are some aspects you can improve on :
 - Try to position your content at the center.
 - Adjust the spacing between the elements to match the original design
 
image

 - Adjust the padding of the language elements to match the original design
 
image

File: index.html [hunk 2]
```diff
@@ -41,6 +41,210 @@ <h2>I'm a software developer</h2>
...
+        <div class="text2">
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has ben the industry's standard dummy text ever
+            since the 1500s
+          </p>
+        </div>
+        <div class="projlang">
+          <ul>
+            <li><a href="index.html">HTML</a></li>
```
Comment (ghost) [inline index.html:85]: - [ ] Your language cards appear to be slightly different from the original design. Kindly adjust them

image

###  Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @cosywasswa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @cosywasswa 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Proper linter check ✅ 
- Changes recommended followed  👍

## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.



Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**